configure time error when evaluating MPI

**Describe the bug**
I tried to build GEOPM and I expected ```./configure``` to succeed.  Instead it fails.

**GEOPM version**
cacfa817b

**Error messages produced**
```
configure: error: Failed to determine MPI C build flags use --with-mpi-bin or --with-mpicc or --disable-mpi
```

**Additional context**
An excerpt from config.log:
```
conftest.c:13:10: fatal error: 'mpi.h' file not found
```

Looking at the arguments passed to the compiler, double quotes were injected around any option that was specified as a path (so anything passed to -I and -L).  These double quotes seem to break this linking check done in m4/lx_find_mpi.m4.

Potentially related to UTF-8 vs. unicode double quotes.

Note that the output from ```mpiicc -show``` emits double quotes.

--- a/integration/config/smng_env.sh
+++ b/integration/config/smng_env.sh
@@ -11,20 +11,28 @@
 
 # Default Intel Toolchain compiler overrides
 # The MPI compiler wrappers are supported by Intel, but are not Intel specific.
-module swap intel intel/19.1
-module load python/3.6_intel
 unset OMP_NUM_THREADS # Set to 1 by a default lmod module
 
 # One time setup:
 # python -m pip install --user -U pip
 # python -m pip install --user --ignore-installed -r geopm/scripts/requirements.txt
+# python -m pip install --user --ignore-installed -r geopm/service/requirements.txt
 
+export CC=icc
+export CXX=icpc
+export FC=ifort
+export F77=ifort
+export F90=ifort
+export MPICC=mpiicc
+export MPICXX=mpiicpc
 export MPIFORT=mpiifort
 export MPIFC=mpiifort
 export MPIF77=mpiifort
 export MPIF90=mpiifort
 
-export GEOPM_USER_ACCOUNT=h1906
+export CPATH=${MKL_INCDIR}:${MKL_F90_INC}:${CPATH} # So that mkl.h can be found
+
+# export GEOPM_USER_ACCOUNT=h1906
 export GEOPM_SYSTEM_DEFAULT_QUEUE=test
 export GEOPM_SBATCH_EXTRA_LINES="
 #SBATCH --constraint=\"noscratch\"

--- a/m4/lx_find_mpi.m4
+++ b/m4/lx_find_mpi.m4
@@ -190,6 +190,7 @@ AC_DEFUN([LX_QUERY_MPI_COMPILER],
      if [[ ! -z "$lx_mpi_command_line" ]]; then
          # Now extract the different parts of the MPI command line.  Do these separately in case we need to
          # parse them all out in future versions of this macro.
+         lx_mpi_command_line=`echo "$lx_mpi_command_line" | sed s/\"//g`
          lx_mpi_defines=`    echo "$lx_mpi_command_line" | grep -o -- '\(^\| \)-D\([[^\"[:space:]]]\+\|\"[[^\"[:space:]]]\+\"\)'`
          lx_mpi_includes=`   echo "$lx_mpi_command_line" | grep -o -- '\(^\| \)-I\([[^\"[:space:]]]\+\|\"[[^\"[:space:]]]\+\"\)'`
          lx_mpi_link_paths=` echo "$lx_mpi_command_line" | grep -o -- '\(^\| \)-L\([[^\"[:space:]]]\+\|\"[[^\"[:space:]]]\+\"\)'`

--- a/service/src/MSRIOGroup.cpp
+++ b/service/src/MSRIOGroup.cpp
@@ -1559,7 +1559,17 @@ namespace geopm
         check_top_level(root);
 
         auto msr_obj = root["msrs"].object_items();
-        bool is_trl_writable = is_trl_writable_in_all_domains(root["msrs"], m_platform_topo, m_msrio);
+        bool is_trl_writable = false;
+        try {
+            is_trl_writable = is_trl_writable_in_all_domains(root["msrs"], m_platform_topo, m_msrio);
+        }
+        catch (const Exception &ex) {
+#ifdef GEOPM_DEBUG
+            std::cerr << "Warning: <geopm> MSRIOGroup::" << std::string(__func__)
+                      << "(): Unable to check TRL via PLATFORM_INFO: "
+                      << ex.what() << std::endl;
+#endif
+        }
 
         for (const auto &msr : msr_obj) {
             std::string msr_name = msr.first;

--- a/service/src/MSRIOGroup.cpp
+++ b/service/src/MSRIOGroup.cpp
@@ -148,6 +148,11 @@ namespace geopm
         }
         catch (const Exception &ex) {
             // Only load architectural MSRs
+#ifdef GEOPM_DEBUG
+            std::cerr << "Warning: <geopm> MSRIOGroup::MSRIOGroup(): "
+                      << "Unable to load CPUID specific MSRs: "
+                      << ex.what() << std::endl;
+#endif
         }
         auto custom_files = msr_data_files();
         for (const auto &filename : custom_files) {

--- a/tutorial/admin/00_test_prereqs.sh
+++ b/tutorial/admin/00_test_prereqs.sh
@@ -43,7 +43,7 @@ if ldd $(which geopmctl) | grep --quiet "not found"; then
 fi
 
 # Python reqs
-PYTHON_CMD=python
+PYTHON_CMD=python3
 if ! 2>/dev/null ${PYTHON_CMD} -c "import geopmpy"; then
     echo "ERROR: geopmpy is not in the PYTHONPATH."
     echo "       Please read the \"USER ENVIRONMENT\" section of the main README."

